corehq/apps/smsbillables/migrations/0010_add_index_to_log_id.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+
+        # Changing field 'SmsBillable.gateway_fee'
+        db.alter_column(u'smsbillables_smsbillable', 'gateway_fee_id', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['smsbillables.SmsGatewayFee'], null=True, on_delete=models.PROTECT))
+        # Adding index on 'SmsBillable', fields ['log_id']
+        db.create_index(u'smsbillables_smsbillable', ['log_id'])
+
+
+        # Changing field 'SmsBillable.usage_fee'
```

Comment:
Yeah we should remove this extra stuff. I think at this point it might be easier to just start fresh by copying the previous migration file (0009), and just insert into it only the create_index and delete_index calls for forwards and backwards. And then just update the frozen model with 'db_index': 'True' for this field.